openedx_lti_tool_plugin/views.py

```diff
@@ -303,6 +320,7 @@ def get(
         # to enable this optimizations from other addresses.
         request.META['HTTP_REFERER'] = getattr(settings, 'LEARNING_MICROFRONTEND_URL', '')
 
+        # verify if check is enrolled is working if not add is enrolled method
```

Comment:
@alexjmpb I tested this and it's working, it raises a 404 response. Do we need to have this comment?